api/pkg/s3/bucketdelete.go

```diff
@@ -16,34 +16,27 @@ package s3
 
 import (
 	"net/http"
+
 	"github.com/emicklei/go-restful"
-	log "github.com/sirupsen/logrus"
 	"github.com/opensds/multi-cloud/api/pkg/common"
-	. "github.com/opensds/multi-cloud/s3/pkg/exception"
+	"github.com/opensds/multi-cloud/s3/error"
 	"github.com/opensds/multi-cloud/s3/proto"
+	log "github.com/sirupsen/logrus"
 )
 
 func (s *APIService) BucketDelete(request *restful.Request, response *restful.Response) {
 	bucketName := request.PathParameter("bucketName")
-	log.Infof("Received request for bucket details: %s", bucketName)
+	log.Infof("Received request for deleting bucket[name=%s].\n", bucketName)
```

Comment:
Here we are changing the logic..Is this fine? Why not warn the user that there are some objects in the bucket?

Reply:
thanks for your suggestion, the logic will be moved into s3 service layer. so, api layer only contains the single call to s3 service. So, i think it is fine in api layer.